### ISSUE TITLE
Provide a means to configure the URL the internal haproxy-exporter uses 

### ISSUE THREAD

**Issue by je-al** [vfarcic/docker-flow-proxy#323]
Hi, I've been testing the metrics endpoint and ran into an issue. Since I'm using a modified image which setups a separate frontend for the stats:
```
frontend stats
    bind *:81
    default_backend stats
backend stats
    mode http
{{.Stats}}
```
It'd be nice if a means to configure the URL the exporter uses internally to fetch the stats was available (preferably through secrets, since this would include authentication if enabled)

**Comment by vfarcic** [comment]
This issue is fixed and available as tag `17.09.06-6`. DFP will respect the value of the environment variable `STATS_URI`. Metrics retrieval is still hard-coded to the port `80`. This will be changed with the issue #324.

**Comment by vfarcic** [comment]
The implementation passed automated tests but I did not have time to try it out on a functional level. Can you please give me a hand and test the tag `beta` and report back whether it works as expected? You'll need to define environment variable `STATS_PORT`.
